### DISCUSSION TITLE
Is there exists a way to add a global options, e.g. {changed: true}  to record.save method?

### DISCUSSION THREAD

**Discussion by zw963** [jeremyevans/sequel#1805] [General]
For my usage case:

on my table, exists a stored generated column,  which update automatically when another columns
get updated, if not use this option, always get following error:

```
Caused by PG::SyntaxError: ERROR:  column "textsearchable_index_col" can only be updated to DEFAULT
```

**Comment by jeremyevans** [comment]
You can use the skip_saving_columns plugin for this: http://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/SkipSavingColumns.html

**Reply by zw963** [reply]
It works! thank you!
